addons/mail/static/tests/owl/form_renderer_chatter_tests.js

```diff
@@ -63,6 +63,276 @@ QUnit.test('basic chatter rendering', async function (assert) {
     );
 });
 
+QUnit.test('basic chatter rendering without followers', async function (assert) {
+    assert.expect(7);
+    this.data['res.partner'] = {
+        fields: {
+            message_attachment_count: {
+                string: 'Attachment count',
+                type: 'integer'
+            },
+            activity_ids: {
+                string: "Activities",
+                type: 'one2many',
+                relation: 'mail.activity'
+            },
+            message_ids: {
+                string: "Messages",
+                type: 'one2many',
+                relation: 'mail.message'
+            },
+            message_follower_ids: {
+                string: "Followers",
+                type: 'one2many',
+                relation: 'mail.followers'
+            },
+            name: {string: "Name", type: 'char'},
+        },
+        records: [{
+            activity_ids: [],
+            id: 2,
+            display_name: "second partner",
+            message_ids: [],
+            message_follower_ids: [],
+        }],
+    };
+    await this.createView({
+        data: this.data,
+        hasView: true,
+        // View params
+        View: FormView,
+        model: 'res.partner',
+        arch: `<form string="Partners">
```

Comment:
Move tag to new line to keep indent with closing tag (same for other tests), similar to what was done for templates in the component.